pkg/asset/machines/machineconfig/ipv6.go

```diff
@@ -0,0 +1,42 @@
+package machineconfig
+
+import (
+	"fmt"
+
+	igntypes "github.com/coreos/ignition/v2/config/v3_2/types"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/openshift/installer/pkg/asset/ignition"
+	mcfgv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
+)
+
+// ForIPv6Addresses creates the MachineConfig to tell kernel to configure the IP addresses with DHCP and DHCPV6.
+func ForIPv6Addresses(role string) (*mcfgv1.MachineConfig, error) {
```

Comment:
Shouldn't it be `ForDualStack()` then? Because the current name `ForIPv6Addresses()` and the name of the file suggest it configures the node for IPv6 only.